backend/src/govsg/services/govsg-template.service.ts

```diff
@@ -27,6 +27,22 @@ export async function getFilledTemplate(
   return pivot.govsgTemplate
 }
 
+const getLocalisedTemplateBody = (
```

Comment:
Will only think of moving this to a separate file if more functions are using this. Right now, only 1 usage.